homeassistant/components/abode/manifest.json

```diff
@@ -5,5 +5,6 @@
   "documentation": "https://www.home-assistant.io/integrations/abode",
   "requirements": ["abodepy==0.18.1"],
   "dependencies": [],
-  "codeowners": ["@shred86"]
+  "codeowners": ["@shred86"],
+  "quality_scale": "gold"
```

Comment:
We need at least 95% coverage on average and all modules tested. What's the current status?

Reply:
All modules are tested. Everything is 100% except init.py (91%), camera.py (66%) and sensor.py (98%). The camera module is obviously bringing the average down. I'm looking at camera.py now and it looks like it may need to be reworked. The `get_image` method is making a [`requests.get`](https://github.com/home-assistant/core/blob/f9634f023297a76d1c523875c5592c355d6b9bf4/homeassistant/components/abode/camera.py#L67) directly from the integration which my understanding is not desired (should be occurring in the 3rd party API). I can't seem to find any [documentation](https://developers.home-assistant.io/docs/entity_index) on how cameras are implemented in HA so I'm probably going to need some time to see if I can figure this out make some modifications to both abodepy and the HA abode camera integration.

Reply:
I think the frontend is calling `camera_image` every 10 seconds when viewing the camera card in the frontend, if streaming is not activated.

Reply:
I think we should remove the gold status for now and merge this PR. Make another PR to complete tests and get gold status.

Reply:
Sounds good to me. Is the 95% coverage a requirement for gold or silver? 

We ought to update the documents as well since it doesn't mention anything about the amount of coverage.

Reply:
Just for gold level.